src/go/src/ebw/www/repoHandlers.go

```diff
@@ -136,10 +136,20 @@ func repoDetails(c *Context) error {
 	c.D[`UserName`] = client.Username
 	c.D[`RepoOwner`] = repoOwner
 	c.D[`RepoName`] = repoName
-	c.D[`RepoFiles`], err = git.ListAllRepoFiles(client, client.Username, repoOwner, repoName)
+
+	repoDir, err := git.RepoDir(client.Username, repoOwner, repoName)
 	if nil != err {
 		return err
 	}
+
+	if(len(prs) > 1) {
+		//What commit hash do we need to use here?
+		val, err := git.ListChangedFiles(client, prs[0].Base.GetSHA(), repoDir, client.Username, repoOwner, repoName)
```

Comment:
Hmm... I'm not 100% sure, but I think I would look at the incoming PR's Head and Base values at https://godoc.org/github.com/google/go-github/github#PullRequest . These have a GetSHA() method, and I think that will give you the SHA you need... but you probably want to compare the HEAD (ie HEAD of the PR) against the HEAD of the current branch on repo... although we're only interested in the changes on PR HEAD...